internal/cli/kraft/cloud/quotas/quotas.go

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2023, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+
+package quotas
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/config"
+	"kraftkit.sh/internal/cli/kraft/cloud/utils"
+	"kraftkit.sh/log"
+
+	kraftcloud "sdk.kraft.cloud"
+)
+
+type QuotasOptions struct {
+	Output string `local:"true" long:"output" short:"o" usage:"Set output format" default:"table"`
+
+	metro string
+}
+
+func NewCmd() *cobra.Command {
+	cmd, err := cmdfactory.New(&QuotasOptions{}, cobra.Command{
+		Short:   "View your resource quota on KraftCloud",
+		Use:     "quotas",
+		Aliases: []string{"q", "quota"},
+		Annotations: map[string]string{
+			cmdfactory.AnnotationHelpGroup: "kraftcloud-quota",
+		},
```

Comment:
No need for its own group, this can be removed:
```diff
- 		Annotations: map[string]string{
- 			cmdfactory.AnnotationHelpGroup: "kraftcloud-quota",
- 		},
```